### DISCUSSION TITLE
UNO app package conflict.

### DISCUSSION THREAD

**Discussion by ApoorvaPetkar** [unoplatform/uno#9356] [General]
Hi, 

I created app using UNO template provided in the documentation. I saw that the project is using windows app SDK internally and is referring Microsoft.UI.Xaml package to get the controls.  And I'm following Windows app development documentation( https://docs.microsoft.com/en-us/windows/apps/design/controls/navigationview.) to add these controls to my project. However I'm facing following issue while I'm trying to code - Not able to reference NavigationView control of Microsoft.UI.Xaml controls.  However if I remove muxc namespace tag from below code and run, it works. 

Not sure is it using winui3 or platform specific version internally.  And I want to use win library-based controls.

My project code-

Followed ditto step from doc- https://platform.uno/docs/articles/get-started-vs-2022.html?tabs=windows, for setting up the project and made following changes in existing files. 

MainPage.xaml

Added  xmlns:muxc="using:Microsoft.UI.Xaml.Controls" 
```xaml
<Page
    x:Class="Test.MainPage"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:Test"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
    mc:Ignorable="d"    
    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">

    <Grid>
        <muxc:NavigationView>
            <muxc:NavigationView.MenuItems>
                <muxc:NavigationViewItem Content="Home" Icon="Home" ToolTipService.ToolTip="Home"/>
                <muxc:NavigationViewItem Content="Collections" Icon="Keyboard" ToolTipService.ToolTip="Collections">
                    <muxc:NavigationViewItem.MenuItems>
                        <muxc:NavigationViewItem Content="Notes" Icon="Page" ToolTipService.ToolTip="Notes"/>
                        <muxc:NavigationViewItem Content="Mail" Icon="Mail" ToolTipService.ToolTip="Mail"/>
                    </muxc:NavigationViewItem.MenuItems>
                </muxc:NavigationViewItem>
            </muxc:NavigationView.MenuItems>
        </muxc:NavigationView>
    </Grid>
</Page>
```

App.xaml 
```xaml
<Application
    x:Class="Test.App"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:local="using:Test">
	
    <Application.Resources>
        <ResourceDictionary>
            <ResourceDictionary.MergedDictionaries>
                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
                <!-- Add resource dictionaries here -->                
            </ResourceDictionary.MergedDictionaries>
            <!-- Add resources here -->
        </ResourceDictionary>
    </Application.Resources>

</Application>
```

Also I tried to add new UNO platform page from add button in visual studio solution it is referring windows.ui.xaml however while creating project it is using microsoft.ui.xaml as you see in the App.xaml file. 

Any pointers would be really helpful to find where I'm going wrong.  Thank you!

**Comment by Paolo55555** [comment]
I guess you selected the "Page (Uno Platform UWP)" template. Maybe you selected the "Uno Platform" filter in the "Add New Item" window. In this case, due to some bug, the only templates shown are the UWP ones. The WinUI templates appear only if you remove the filter. You should select the "Page (Uno Platform WinUI 3)" template.

**Reply by ApoorvaPetkar** [reply]
Yes. Thanks @Paolo55555
